component/input.css

```
@@ -0,0 +1,66 @@
+.form-style-9{
+	max-width: 450px;
+	background: #FFFFFF;
+	padding: 30px;
+	margin: 50px auto;
+	box-shadow: 1px 1px 15px rgba(0, 0, 0, 0.125);
+	border-radius: 10px;
+    border: 1px solid rgba(0,0,0,.125);
+}
+.form-style-9 ul{
+	padding:0;
+	margin:0;
+	list-style:none;
+}
+.form-style-9 ul li{
```

```suggestion
.form-style-item{
```
- Avoid such deep nesting, instead use the class names inside the child items.
- Deep nesting increases the specificity of that element which will cause you trouble later.

- It's good practice to keep the specificity of any element as low as possible.
- For Ref: https://dev.to/emmabostian/css-specificity-1kca